Sources/Mock.swift

```diff
@@ -99,6 +99,9 @@ public struct Mock: Equatable {
     /// Add a delay to a certain mock, which makes the response returned later.
     public var delay: DispatchTimeInterval?
 
+    /// Allow response cache
```

Comment:
Very, verrrry NIT but, you know

```suggestion
    /// Allow response cache.
```